framework/postcommit_tasks/handlers.py

```diff
@@ -35,12 +48,20 @@ def postcommit_after_request(response, base_status_error_code=500):
             for func in postcommit_queue().values():
                 pool.spawn(func)
             pool.join(timeout=5.0, raise_error=True)  # 5 second timeout and reraise exceptions
+
+        if postcommit_celery_queue():
+            if settings.USE_CELERY:
+                postcommit_celery_task_wrapper.delay(postcommit_celery_queue())
```

Comment:
This is the top-secret bad way to get around the maximum recursion bug. Make a task that calls all the other tasks as a chain and call that, passing in the signatures of the other tasks. I read about it on the internet.